lib/mongoid/association/relatable.rb

```diff
@@ -323,6 +323,12 @@ def validate?
 
       private
 
+      # Gets the model classes with inverse associations of this model. This is used to determine
+      # the classes on the other end of polymorphic relations with models.
```

Comment:
`@api private` please.

Reply:
Just to clarify, do you mean that this should be made public but marked `@api private`, or that I should just add that to the method where it is?

Reply:
I missed the private access specifier, I suppose this is fine as it is then.